ee/experimental/incident_utils.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
```

Comment:
Generally, this file is under-documented, it is hard to understand how/what the functions are doing